test/sync.js

```diff
@@ -46,9 +46,10 @@ test('file exits', t => {
 });
 
 test('root dir', t => {
+	const mode = fs.statSync('/').mode & 0o777;
```

Comment:
Can you move this to the `assertDir` function?

Reply:
Do you mean to have `assertDir` just skip the `fs.statSync` check if dir === '/'?  The `root dir` test in my patch verifies that `make-dir` doesn't modify the mode of an existing directory.

Reply:
Ah, nevermind. I misunderstood the usage.